Parkinson.xcodeproj/project.pbxproj

```diff
@@ -944,7 +988,7 @@
 				HEADER_SEARCH_PATHS = (
 					"$(inherited)",
 					/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include,
-					"$(PROJECT_DIR)/PDScores/PDScores/**",
+					"$(PROJECT_DIR)/Parkinson/Library/PDScores/PDScores/**",
```

Comment:
Search path removed - leftover prior to PDScores refactored into a standalone project.